src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -741,6 +745,7 @@ type FrameworkIdentifier =
         | DotNetFramework FrameworkVersion.V4_7_2 -> [ DotNetFramework FrameworkVersion.V4_7_1 ]
         | DotNetFramework FrameworkVersion.V4_8 -> [ DotNetFramework FrameworkVersion.V4_7_2 ]
         | DotNetFramework FrameworkVersion.V5 -> [ DotNetCoreApp DotNetCoreAppVersion.V3_1; DotNetStandard DotNetStandardVersion.V2_1 ]
+        | DotNetFramework FrameworkVersion.V6 -> [ DotNetCoreApp DotNetCoreAppVersion.V3_1; DotNetStandard DotNetStandardVersion.V2_1 ]
```

Comment:
I think net6.0 should point to net5.0 instead of .netcoreapp3.1 for the direct dependency tree